server/src/main/java/org/eclipse/lsp/cobol/service/Configuration.java

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2022 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *    Broadcom, Inc. - initial API and implementation
+ *
+ */
+package org.eclipse.lsp.cobol.service;
+
+import org.eclipse.lsp.cobol.core.model.tree.EmbeddedCodeNode;
+
+import java.util.List;
+import java.util.Set;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.Consumer;
+
+/** This interface handles the request for configurations from the client settings */
+public interface Configuration {
+
+  /**
+   * Get all the configurations from the client Settings
+   *
+   * @return {@link Consumer < List <Object>>}
+   */
+  CompletableFuture<Void> getConfigurationFromSettings();
```

Comment:
Why are you returning the Future here? It is not used anywhere. It can be just void.

Reply:
And since you don't care about the return value and the main reason for the method to update `Configuration` internal state it can be named like `updateConfigurtionFromSettings`.